lib/routes/callback/index.js

```diff
@@ -5,6 +5,41 @@ const { TooManyRequests } = require('http-errors')
 const { metricsInsert } = require('../metrics/query')
 const { registerUpdate } = require('./query')
 
+/**
+ * Allows users to request a callback from a Contact Tracer. If accepted, the request is added to the
+ * AWS callback queue with information about the user to call. The user's phone # is submitted as
+ * part of this request, but the phone # IS NOT stored with the user's database record. It is included
+ * in the body of the AWS callback queue message so that Contact Tracers know which number to call.
+ *
+ * Note that within the AWS callback queue the user's ID and phone # are stored together. This is a
+ * potential PII issue as it provides an identifiable mapping between users and their phone numbers.
+ *
+ * See schema.js/callback for details on what data can be submitted with this request.
+ *
+ * Users are identified by the 'id' in the claims of the request Authorization JWT header. The id value
+ * comes from the registration that occurred when the user installed the application.
+ *
+ * When the request comes in the user's registration record will be updated setting the last_callback
+ * column to the current timestamp IFF the last_callback column is empty. This prevents the same user
+ * from entering multiple callback requests.
+ *
+ * Not verified, but I assume the process that Contact Tracers use will clear the last_callback value
+ * of the user's registration record thus allowing users to request another call should they need it.
+ *
+ * From what I can tell in the app, callbacks can be requested if the app determines the user was in
+ * close contact with some exposure key (ie, a person with a positive diagnosis). Until the user gets
+ * an actual callback the last_callback cell will have a value so the user will not be able to request
+ * another callback. But, if the user has gotten a callback and the cell is set to NULL then the user
+ * will be able to request another callback.
+ *
+ * All requests go through authentication which means both the user and device will be verified. This
+ * process will prevent random people from submitting callback requests. Theoretically, only the app
+ * can make this call.
+ *
+ * Responses:
+ *  204: The callback request was successfully accepted and written to the database
+ *  429: An existing unfulfilled callback already exists for the user
```

Comment:
The `last_callback` is currently not cleared, in Ireland the requirement was one callback only per person. Clearing the last_callback would allow a subsequent callback to be queued for the same person

Reply:
Have updated the docs to indicate this.